### ISSUE TITLE
SplitButton: Insufficient accessibility suitability, operation with screen reader not possible

### ISSUE THREAD

**Issue by Rene-Stoeckel** [primefaces/primefaces#13538]
### Description

During our accessibility assessment of our application the SplitButtons were criticised as a last problem.
Although normal usability via keyboard is completely fine, it is insufficient when screenreaders (JAWS or NVDA)
are used: 
It is possible to open the split button with the screenreader navigation, but it is impossible 
to navigate to the subelement buttons. Also the screenreaders do not properly read the elements and subelements of the split button.

### Describe the solution you would like

The testers analyzed the showcase:
https://www.primefaces.org/showcase/ui/button/splitButton.xhtml?jfwid=61a87
and gave some details and hints with a screenshot, which I attached.

![Image](https://github.com/user-attachments/assets/98c97008-2182-42f4-bee3-bf968725d60c)

The attribute „aria-haspopup“ is on the wrong html level. It should be moved to the affected interactive element, in
this case the second, highlighted <button> element in the screenshot, because this has the popup. It's value should not be "true" but "menu" for this situation.

The dropdown button has an improper name "ui-button". For usage with screenreader it should be adjustable, e.g. to contain a text like "more options", or "more actions."

The testers did not give us more recommendations how to finally fix the split button,
instead they pointed to an example how such a split button should be 
implemented to be accessibility compliant:
https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-links/

### Additional context

We have a workaround: 
We used standard buttons for the actions as we only grouped 2 or 3 buttons with the split button and we had enough space in our tool bar.

**Comment by Rene-Stoeckel** [comment]
Yes, sure, we'll take a look, thank you very much for supporing it! Is there a way to switch the showcase from v14.0.6-SNAPSHOT to 15.0.2? Otherwise we need to upgrade to 15 which is not instant.
